crates/superposition_types/src/lib.rs

```diff
@@ -74,3 +77,276 @@ impl FromRequest for User {
         }
     }
 }
+
+#[derive(Debug, Clone, PartialEq)]
```

Comment:
can we use something like this:
```suggestion
#[derive(Debug, Clone, PartialEq)]
pub enum ValidationType {
    DEFAULT,
    EXPERIMENTAL,
    #[cfg(feature = "db_type_validation_disabled")]
    DB,
}

pub fn get_db_validation_type() -> ValidationType {
    #[cfg(feature = "db_type_validation_disabled")]
    return ValidationType::DB;
    #[cfg(not(feature = "db_type_validation_disabled"))]
    return ValidationType::DEFAULT;
}
```

we can utilise rust features here, and anyone who has old version of code can go ahead with `db_type_validation_disabled` feature (or whatever you want to call it) as newer versions will not have discrepancy in DB data

wherever you want to use ValidationType::DB, just use this function `get_db_validation_type` directly, which will resolve the value based on the feature which the user of superposition requires

later on, when we are sure that data is not longer corrupt in DB, we can get the feature removed altogether

Reply:
places having a switch case can use something like:
```
            #[cfg(feature = "db_type_validation_disabled")]
            ValidationType::DB => (),
```